numpyro/contrib/hsgp/approximation.py

```diff
@@ -7,6 +7,8 @@
 
 from __future__ import annotations
 
+from typing import cast
```

Comment:
maybe using `jnp.asarray(...)` instead of cast

Reply:
changed in https://github.com/pyro-ppl/numpyro/pull/1940/commits/ebc510c2aac444abe70246cb869fa835a7837a64